code/modules/vending/_vending.dm

```diff
@@ -59,6 +59,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	payment_department = ACCOUNT_SRV
 	light_power = 0.5
 	light_range = MINIMUM_USEFUL_LIGHT_RANGE
+	tk_throw_range = 0
```

Comment:
This shouldn't be necessary? I meant remove it if it existed on the vending machine type.